packages/components/src/AppHeaderBar/__snapshots__/AppHeaderBar.snapshot.test.js.snap

```diff
@@ -386,12 +386,14 @@ exports[`Storyshots AppHeaderBar while searching 1`] = `
                 autoComplete="off"
                 autoFocus={true}
                 className="form-control"
+                disabled={undefined}
                 id="react-autowhatever-42-input"
                 onBlur={[Function]}
                 onChange={[Function]}
                 onFocus={undefined}
                 onKeyDown={undefined}
                 placeholder={undefined}
+                readOnly={undefined}
```

Comment:
Please make sure that `disabled` and `readOnly` have default values rather than being undefined.